packages/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tree-house/core",
-  "version": "2.0.0",
+  "version": "2.1.0-rc.1",
```

Comment:
Made it an rc as we need to test it on an actual deployed API service since it sometimes acts weird on local environment vs. deployed kubernetes cluster